### PR TITLE
editing support link

### PR DESCRIPTION
**Pull request by fiquick**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-329.surge.sh

Updated pages:
- [Neo4j AuraDB overview](https://neo4j-docs-aura-329.surge.sh/aura/auradb)
- [Overview](https://neo4j-docs-aura-329.surge.sh/aura/platform/api/overview)
- [Security](https://neo4j-docs-aura-329.surge.sh/aura/platform/security)
- [Troubleshooting](https://neo4j-docs-aura-329.surge.sh/aura/tutorials/troubleshooting)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-329.surge.sh

Updated pages:
- [Neo4j AuraDB overview](https://neo4j-docs-aura-329.surge.sh/aura/auradb)
- [Overview](https://neo4j-docs-aura-329.surge.sh/aura/platform/api/overview)
- [Security](https://neo4j-docs-aura-329.surge.sh/aura/platform/security)
- [Troubleshooting](https://neo4j-docs-aura-329.surge.sh/aura/tutorials/troubleshooting)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-329.surge.sh

Updated pages:
- [Neo4j AuraDB overview](https://neo4j-docs-aura-329.surge.sh/aura/auradb)
- [Overview](https://neo4j-docs-aura-329.surge.sh/aura/platform/api/overview)
- [Security](https://neo4j-docs-aura-329.surge.sh/aura/platform/security)
- [Troubleshooting](https://neo4j-docs-aura-329.surge.sh/aura/tutorials/troubleshooting)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-329.surge.sh

Updated pages:
- [Neo4j AuraDB overview](https://neo4j-docs-aura-329.surge.sh/aura/auradb)
- [Overview](https://neo4j-docs-aura-329.surge.sh/aura/platform/api/overview)
- [Security](https://neo4j-docs-aura-329.surge.sh/aura/platform/security)
- [Troubleshooting](https://neo4j-docs-aura-329.surge.sh/aura/tutorials/troubleshooting)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-329.surge.sh

Updated pages:
- [Neo4j AuraDB overview](https://neo4j-docs-aura-329.surge.sh/aura/auradb)
- [Overview](https://neo4j-docs-aura-329.surge.sh/aura/platform/api/overview)
- [Security](https://neo4j-docs-aura-329.surge.sh/aura/platform/security)
- [Troubleshooting](https://neo4j-docs-aura-329.surge.sh/aura/tutorials/troubleshooting)

File: modules/ROOT/pages/platform/api/overview.adoc [hunk 1]
```diff
@@ -52,4 +52,4 @@ An `X-Request-Id` response header is returned with each request and can be used
 
 The value of this header contains a unique ID that can be used to track the journey of a request.
 
-If you run into any issues with a particular request, you can https://aura.support.neo4j.com/hc/en-us/[raise a support ticket] and provide the `X-Request-Id`.
\ No newline at end of file
+If you run into any issues with a particular request, you can https://support.neo4j.com/hc/en-us/requests/new[raise a support ticket] and provide the `X-Request-Id`.
```
Comment (nvitucci) [inline modules/ROOT/pages/platform/api/overview.adoc:55]: Let's just use https://support.neo4j.com/.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-329.surge.sh

Updated pages:
- [Neo4j AuraDB overview](https://neo4j-docs-aura-329.surge.sh/aura/auradb)
- [Overview](https://neo4j-docs-aura-329.surge.sh/aura/platform/api/overview)
- [Security](https://neo4j-docs-aura-329.surge.sh/aura/platform/security)
- [Troubleshooting](https://neo4j-docs-aura-329.surge.sh/aura/tutorials/troubleshooting)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-329.surge.sh

Updated pages:
- [Neo4j AuraDB overview](https://neo4j-docs-aura-329.surge.sh/aura/auradb)
- [Overview](https://neo4j-docs-aura-329.surge.sh/aura/platform/api/overview)
- [Security](https://neo4j-docs-aura-329.surge.sh/aura/platform/security)
- [Troubleshooting](https://neo4j-docs-aura-329.surge.sh/aura/tutorials/troubleshooting)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-329.surge.sh

Updated pages:
- [Neo4j AuraDB overview](https://neo4j-docs-aura-329.surge.sh/aura/auradb)
- [Overview](https://neo4j-docs-aura-329.surge.sh/aura/platform/api/overview)
- [Security](https://neo4j-docs-aura-329.surge.sh/aura/platform/security)
- [Improving Cypher performance](https://neo4j-docs-aura-329.surge.sh/aura/tutorials/performance-improvements)
- [Troubleshooting](https://neo4j-docs-aura-329.surge.sh/aura/tutorials/troubleshooting)

**Review by nvitucci — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-329.surge.sh

Updated pages:
- [Neo4j AuraDB overview](https://neo4j-docs-aura-329.surge.sh/aura/auradb)
- [Overview](https://neo4j-docs-aura-329.surge.sh/aura/platform/api/overview)
- [Security](https://neo4j-docs-aura-329.surge.sh/aura/platform/security)
- [Improving Cypher performance](https://neo4j-docs-aura-329.surge.sh/aura/tutorials/performance-improvements)
- [Troubleshooting](https://neo4j-docs-aura-329.surge.sh/aura/tutorials/troubleshooting)

File: modules/ROOT/pages/tutorials/troubleshooting.adoc [hunk 2]
```diff
@@ -59,7 +59,7 @@ image::planSummary.png[]
 [NOTE]
 ====
 If this error occurs while loading data from CSV files, use `apoc.periodic.iterate` to import the data and use a relatively small number for the `batch_size` parameter.
-Read more information about link:https://aura.support.neo4j.com/hc/en-us/articles/1500012376402-Using-apoc-to-conditional-loading-large-scale-data-set-from-JSON-or-CSV-files[Using apoc to conditional loading large scale data set from JSON or CSV files].
+For more information, visit the link:https://support.neo4j.com/s/article/1500012376402-Using-apoc-to-conditional-loading-large-scale-data-set-from-JSON-or-CSV-files[Customer Support Portal].
```
Comment (nvitucci) [inline modules/ROOT/pages/tutorials/troubleshooting.adoc:62]:
```suggestion
For more information, see the link:https://support.neo4j.com/s/article/1500012376402-Using-apoc-to-conditional-loading-large-scale-data-set-from-JSON-or-CSV-files[related article] on the Customer Support Portal.
```

**Pull request merged**